components/com_finder/tmpl/search/default_result.php

```diff
@@ -115,7 +116,18 @@
                     <?php endforeach; ?>
                     <?php if (count($taxonomy_text)) : ?>
                         <li class="result__taxonomy-item result__taxonomy--<?php echo $type; ?>">
-                            <span><?php echo Text::_(LanguageHelper::branchSingular($type)); ?>:</span> <?php echo implode(',', $taxonomy_text); ?>
+                            <span><?php echo Text::_(LanguageHelper::branchSingular($type)); ?>:</span>
+                            <?php
+                            $text = implode(',', $taxonomy_text);
+                            ?>
+                            <?php if ($type = 'Type') : ?>
```

Comment:
```suggestion
                            <?php if ($type === 'Type') : ?>
```
Shouldn't this be a comparison and not an assignment?

Reply:
With an assignment you get always true and so always execute the code in that if condition, and the human tests will not catch that because according to the testing instructions it works because the testing instructions only cover the test for the fix of the issue but not other aspects of the functionality not being broken.

Reply:
OMG... as i'm resetting on local my environment, i've done the edit directly in GitHub... 
Thanks for review!